lib/pdf/reader/standard_security_handler.rb

```diff
@@ -164,8 +164,8 @@ def make_file_key( user_pass )
       # e) add the file ID
       @buf << @file_id
       # f) if revision > 4 then if encryptMetadata add 4 bytes of 0x00 else add 4 bytes of 0xFF
```

Comment:
This comment no longer reflect the actual state of the code. It seems like if `revision >= 4` and `NOT @encryptMeta`, then always add 4 bytes of `0xFF` and under no circumstances will 4 bytes of `0x00` ever be added.